Minimal_test/DropDownTest.html

```diff
@@ -0,0 +1,67 @@
+<html>
+  <head>
+	  <title>Santiagos Webseite</title>
+    <link rel="stylesheet" href="pico.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mocha/3.2.0/mocha.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mocha/3.2.0/mocha.js"></script>
+    <script>
+      mocha.setup('bdd'); // minimal setup
+    </script>
+    <!-- add chai -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/chai/3.5.0/chai.js"></script>
+    <script>
+      // chai has a lot of stuff, let's make assert global
+      let assert = chai.assert;
+    </script>
+    
+  </head>
+  <body class="container">
+    <h1>Test</h1>
+    <select id="selectToFill"></select>
+    <script>
+      function fillSelectionBox(selectBoxId, values, selectedValue){
+      let selectedIndex;
+      if (!values.includes(selectedValue)){
+        console.error();
+        return null;
+      }
+      if (document.getElementById(selectBoxId) == null){
+        console.error();
+        return null;
+      }
+        for(let i = 0; i < values.length; i++){
+          document.getElementById(selectBoxId).innerHTML = document.getElementById(selectBoxId).innerHTML +("<option " + "value='" +values[i] +"' >" + values[i] + "</option> ");
```

Comment:
hier bitte wie im Task beschrieben:
Zum Erstellen von HTML Elementen nutz bitte die JavaScript-Funktion `createElement`. Das ist sauberer und schneller. Du nutzt dann auch nicht mehr `innerHTML` sondern andere Funktionen zum Hinzufügen.

Den HTML-Code als String zu übergeben is möglich, aber relativ Fehleranfälllig